vms/platformvm/blocks/executor/proposal_block_test.go

```diff
@@ -184,7 +183,7 @@ func TestBanffProposalBlockTimeVerification(t *testing.T) {
 	env.mockedState.EXPECT().GetTimestamp().Return(chainTime).AnyTimes()
 
 	onParentAccept := state.NewMockDiff(ctrl)
-	onParentAccept.EXPECT().CaminoGenesisState().Return(&genesis.Camino{}, nil)
```

Comment:
Same

Reply:
its required, cause its called during test
and without this line, test will always fail